WebApp/processing/regions.go

```diff
@@ -49,6 +49,36 @@ func (rm *RegionMap) AddPixelToRegion(pixel Pixel, region RegionIndex) {
 	rm.pixels[pixel] = region
 }
 
+func (rm *RegionMap) AddPixelToRegionMap(pixel Pixel, img image.Image) {
+	colorP := img.At(int(pixel.X), int(pixel.Y))
+	regionLeft, hasRegionLeft := rm.pixels[Pixel{pixel.X - 1, pixel.Y}]
+	colorLeft := img.At(int(pixel.X)-1, int(pixel.Y))
+	regionAbove, hasRegionAbove := rm.pixels[Pixel{pixel.X, pixel.Y - 1}]
```

Comment:
for our regionLeft var i see that we decrement x by one. However why in regionAbove do we decrement Y by one ?  Would we rather want 
`regionAbove, hasRegionAbove := rm.pixels[Pixel{pixel.X, pixel.Y - 1}]`

Reply:
Positive Y goes down when you're dealing with images. Funnily enough, positive Y also goes downwards in Godot (at least for 2d projects)